src/mummy/Mummy.ts

```diff
@@ -46,6 +84,11 @@ export default class Mummy extends Phaser.Physics.Arcade.Sprite {
 
     if (this._health <= 0) {
       this.healthState = HealthState.DEAD;
+      this.setTint(0xff0000);
+
+      // TODO: set die animation
```

Comment:
The velocity and animation lines can go since this is done in #preUpdate when the death-animation is over (dt > 250)